.github/workflows/pr-readiness.yml

```
@@ -0,0 +1,273 @@
+            
+            report += `- **File Types**: ${Object.entries(fileTypes).map(([ext, count]) => `${ext} (${count})`).join(', ')}\n`;
+            
+            // Create or update check run
+            const checkName = 'PR Readiness';
+            const { data: checks: checkRuns } = await github.rest.checks.listForRef({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              ref: pr.head.sha,
+              check_name: checkName
+            });
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)